### ISSUE TITLE
A11y: Inverted colors on footer are low-contrast

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#13477]
### User story

**AS A** Veteran using inverted-color tools
**I WANT** the footer to be high-contrast
**SO THAT** I can easily navigate it.

Florence McCafferty came across this during a research session and [reported it to us in Slack](https://dsva.slack.com/archives/C52CL1PKQ/p1682016260320299).

The participant is low-vision and uses a setting on his mobile device to flip the colors, which he was explaining increases the contrast for him, and allows him to better see and read content on his screen. Here's an image of the screen as he was sharing during the interview:
<img width="184" alt="image" src="https://user-images.githubusercontent.com/4054752/234076314-6eeb8fe2-ef7d-441a-9740-cc82487daadf.png">

Verbatim:
“I’m looking at the bottom now, where it’s like bright orange, okay? I don’t know if that’s orange or yellow. That blurs out my contrast. Where it says contact us, Veteran programs and services, that’s really hard to read.”
“As I’m looking at the words I’m only seeing like the first 2 or 3 letters and last 2-3 letters. And where it says Veteran programs and services I had to look a few times to see what it says. I could use better contrast down there.”

### Engineering notes / background
 Brian DeConick provided some info in the slack thread:

I can fill in a few details while Laura is out, but this is definitely worth following up with her on once she's back. The super short version: There might be some CSS improvements that can be made, but it's a little messy.
There are a handful of CSS media queries that can be used to target a user's color preferences: [forced-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors), [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme), [prefers-contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast), and [inverted-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/inverted-colors). Support for each of these varies a ton from browser to browser and device to device.
On my Windows machine, forced-colors styles High Contrast Mode, prefers-color-scheme styles Dark Mode and Light Mode, and I'm not seeing anything different for prefers-contrast or inverted-colors.
On my iPhone, it's the exact opposite --- prefers-contrast works for the Increase Contrast setting and inverted-colors works for the Smart Invert setting. But inverted-colors doesn't work for the Classic Invert setting that's also available in iOS. As far as I can tell, Classic Invert can't be styled at all. Nothing is ever easy. :shrug:
[@Florence McCafferty](https://dsva.slack.com/team/U03HQRVPZD1)
 do you know what device the participant was using? On my iPhone I'm seeing something very similar to the screenshot for both Smart Invert and Classic Invert.
So it may be possible to clean up this user's experience, depending on what device/browser/setting they're using. In general I think it's probably a good idea to test across as many of these combinations as is practical and do some minor style adjustments. But I'll defer to Laura on any recommended action since I don't want to make a ton of extra work for her.

## Additional Notes from Laura
I looked at the colors generated by the "inverted-colors" accessibility display, background is #FFC18C and Text is #000. These produce a contract ratio of 13.26 and with a text size of 16px does pass AAA WCAG guidelines.

The user above is saying that the middle portions of the footer links are not clear and this could perhaps be helped by other style adjustments, not just looking at the contrast.  Some recommendations that we can look at when we redo the footer that may help the user experience of this:

- Increase the size the of the footer links from 16px to 18px
- Increase the letter spacing of the footer links. This doesn't have to be enormous but just a slight increase to make the distinction between the letter more clear

Screenshot of what this might look like:
![image](https://user-images.githubusercontent.com/106678594/235922992-59c4ab6c-ea94-4dba-bbb1-de1c4d44c28f.png)
(I tried to screenshot this inverted but clearly that didn't work)

As far as changing the contrast and choosing color combinations for all the settings Brian mentioned above:
- `inverted-colors`: This seems like the most likely setting that would improve this particular user's experience. However, this is only supported on Safari so I would only be able to test this once in staging/production on my phone
- The other 3 settings mentioned (`prefers-contrast`, `forced-colors`, and `prefers-color-scheme`) are supported by the majority of browsers but we should think about what this would mean - would we be setting CSS for the entire site for these settings? Would we just do the footer since we have a report of a poor user experience with the default color combination?

## Acceptance criteria

- [ ]

**Comment by laflannery** [comment]
@wesrowe I added some notes and comments above. Let me know if you need anything else on this for now or want to discuss further

**Comment by FranECross** [comment]
@laflannery to talk to Amanda K about this item.

**Comment by FranECross** [comment]
@laflannery Just checking to see if you had a chance to chat with Amanda about this... Thanks in advance!

**Comment by laflannery** [comment]
With the new Footer, we have:
- Increased font size from 16px to 19px which increases readability
- Increased letter spacing of the headers on mobile
- Darkened the blue, which inverts to tan (instead of orange/yellow) with black text
  - I still want to find a way to get this exact hex value of the tan
- Added black with white text to the bottom, which inverts to white with black text


All of these updates should provide a better experience for inverted colors, however as mentioned in the ticket description, determining an overall strategy for all browser/device/contrast scenarios is a more difficult and complex task. Completing that full strategy is out of scope for the Header/Footer MVP and should be revisited post-MVP

**Comment by laflannery** [comment]
@FranECross to move to post-mvp tracking

**Comment by laflannery** [comment]
@FranECross @jilladams  I mentioned HCM in my handoff notes that I sent over to the DST. Given the new direction for the Header/Footer project, would this particular ticket be closed now as our team is not going to be updating anything?

**Comment by FranECross** [comment]
Per Dave Conlon, the Federal Header/Footer improvement effort has been closed by the Whitehouse / 18f. All related tickets may be closed.
